implementations/rust/ockam/ockam_api/src/util.rs

```diff
@@ -449,4 +478,22 @@ pub mod test_utils {
             identifier: identity.identifier(),
         })
     }
+
+    async fn create_identity_zero(secure_channels: &Arc<SecureChannels>) -> Result<Identity> {
```

Comment:
What is the reason for creating such Identity?

Reply:
During node manager initiation inside a test, I want to be able to add an authority with a credential.
Since the node manager currently accepts only configuration-based authority parameters, I needed to create the credential **before** the node manager initialization. Hence, I needed to create the same identity twice, the easiest (and most reproducible) way was to create it using a private key made of zeroes.

I'm creating a follow-up to refactor the node creation to remove this hack

Reply:
https://github.com/build-trust/ockam/issues/5118

Reply:
Ok, I think having a pre-generated key as a constant is the better way. I don't want us to have the all-zero private key, even in tests.